### ISSUE TITLE
Project still active?

### ISSUE THREAD

**Issue by AlexanderViand** [Codetector1374/tablegen-intellij#1]
Hi @Codetector1374 ,

I stubmled across this project while looking for lint/autoformat support for (MLIR) tablegen in CLion, and I'm wondering if this is still current/usable :)

**Comment by terryzhao127** [comment]
I wonder that too.

**Comment by Codetector1374** [comment]
Not really. I was working on LLVM backends a while back started this then never really got to finish it :(

**Comment by realqhc** [comment]
I have revived the project and if you check my fork, it can lint most TableGen code in LLVM correctly. https://github.com/realqhc/tablegen-intellij. You can download the compiled artifact from Github Actions and begin using it.
